Runtime/RosMsgs.cs

```diff
@@ -7,6 +7,14 @@ public interface IRosMsg
     {
         public string GetRosType();
     }
+    public class Duration : IRosMsg
+    {
+        string IRosMsg.GetRosType() { return "builtin_interfaces.msg.Duration"; }
```

Comment:
Need to handle ROS1 case.
There are will be little bit different fields and message type. 
msg type is **std_msgs.msg.Duration**
and fields: **secs**,**nsecs**

Reply:
Are the fields secs,nsecs or sec,nsec in ROS1?
![image](https://github.com/user-attachments/assets/eef762fd-57b3-4b2e-83fa-3a7718739b1a)